main/EventSystem.cpp

```diff
@@ -1061,6 +1061,13 @@ void CEventSystem::GetCurrentMeasurementStates()
 				}
 			}
 			break;
+		case pTypeManaged:
+			if (sitem.subType == sTypeManagedCounter)
+			{
+				utilityval = static_cast<float>(atof(sitem.sValue.c_str()));
```

Comment:
This counter only has a counter and no actual usage ? Also not in the feature ?
I say thinking about the feature, there might be someone willing to use this and also report the current usage
So a sValue like COUNTER(number);USAGE(Watt) would be a good idea?